.github/workflows/periodic.yaml

```diff
@@ -0,0 +1,10 @@
+name: periodic
```

Comment:
good idea, however the intent of this job isn't going to be clear to project newbies.  would you mind adding some comments or docs?  also a more descriptive name would help, something like `check-dependencies` or something?

Reply:
sure